backend/app/services/channels/telegram/service.py

```
@@ -0,0 +1,651 @@
+    """
+    from app.models.kind import Kind
+
+    db = SessionLocal()
+    try:
+        channel = (
+            db.query(Kind)
+            .filter(
+                Kind.id == channel_id,
+                Kind.kind == MESSAGER_KIND,
+                Kind.user_id == MESSAGER_USER_ID,
+                Kind.is_active == True,
```

Ruff (0.14.14)

[error] Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)